book/types_of_data.md

```diff
@@ -178,7 +178,7 @@ You can write binary as a literal using the `0x[...]` form:
 
 ## Structured data
 
-Structured data builds from the simple data. For example, instead of a single integer, structured data gives us a way to represent multiple integers in the same value. Here's a list of the currently supported structured data types: rows, lists, and blocks.
+Structured data builds from the simple data. For example, instead of a single integer, structured data gives us a way to represent multiple integers in the same value. Here's a list of the currently supported structured data types: records, lists, tables, blocks and groups.
```

Comment:
I'm not quite sure if this is correct?

Reply:
I'd say `records`, `lists`, and `tables`.

`blocks` doesn't feel quite right, it's code rather than data.
`groups` is a syntax thing rather than a data thing, also.

Reply:
I will update it later